### DISCUSSION TITLE
Keep the selected tab opened when page reloading

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#383] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/211

When I'm developing something that uses the apollo store, I usually have the store tab opened on the devtool to watch the state being update, but everytime I change something on the code and the app reloads, I've to select again the store tab and scroll to the key I was watching.

It would be awesome if the least the tab opened could kept the same between app reloads.

If you think this is a wanted feature, let me know, I would be happy to contribute :)

**Comment by jerelmiller** [comment]
This appears it was fixed in a previous release of the devtools. I can refresh the page while on the Cache tab and see that both the tab and the selection I had remain. I'll go ahead and close this as resolved.
